### ISSUE TITLE
Update mod for game version 1.90

### ISSUE THREAD

**Issue by R3Drosso** [zeobviouslyfakeacc/BreakDownCans#2]
Hi,
could you update your mod to make it compatible with version 1.90 of the game? (or even tell me how to do that)
Thanks!

**Comment by zeobviouslyfakeacc** [comment]
Hi @R3Drosso

This mod has been superseded by [TinCanImprovements](https://github.com/zeobviouslyfakeacc/TinCanImprovements), which works with current versions of TLD. It allows you to craft 1 scrap metal from 4 tin cans, reduces the weight of tin cans to 0.05 kg (based on me weighing a tin can IRL), and fixes a bug in the game where the weight of food that comes in cans could drop below the weight of the can. I thus do not plan to also update BreakDownCans.

**Comment by R3Drosso** [comment]
Hi @zeobviouslyfakeacc 
thank you very much for the reply, I hadn't seen that mod, sorry for the trouble!
May I ask you if it's complicated to update a mod? Is there a guide somewhere?
Because I would like to update mods that interest me, but that are old.
(CarcassMoving 
https://github.com/Deus13/CarcassMoving
and Non-Stroymode-Bearspear for example
https://github.com/Deus13/Non-Stroymode-Bearspear)
Thanks!

**Comment by zeobviouslyfakeacc** [comment]
Yeah, most of the resources you'll need (and the people with the know-how) can be found on [the TLD modding discord server](https://discord.gg/VqCUgTJ).

As for the mods you mentioned, Deus13 has come back to modding recently and has started updating some of their old mods, so perhaps the mods you're looking for will also be updated soon 😃
